src/apm.coffee

```diff
@@ -48,6 +48,34 @@ module.exports =
           programFilesPath = path.join(process.env.ProgramFiles, 'Atom', 'resources', 'app.asar')
           callback(programFilesPath)
 
+  getResourcePathSync: (callback) ->
```

Comment:
I was thinking of working to make `getNodeVersion` actually work properly with async, but I'm also totally okay with using a synchronous version of `getResourcePath` - whichever way you think would work better.